docs/en/latest/building-apisix.md

```diff
@@ -45,24 +45,24 @@ To build and package APISIX for a specific platform, see [apisix-build-tools](ht
 
 ## Building APISIX from source
 
-First of all, we need to specify the version `APISIX_VERSION` to be installed:
+First of all, we need to specify the branch to be built:
 
 ```shell
-APISIX_VERSION='3.11.0'
+APISIX_BRANCH='release/3.11.0'
```

Comment:
It was a bit outdated

<img width="958" alt="image" src="https://github.com/user-attachments/assets/7bcc5bd3-733c-46ad-94ab-42f4758c63e7">

<img width="339" alt="image" src="https://github.com/user-attachments/assets/94387261-ed14-413b-9c9d-91d2b20d5021">